python3.8-image/cookiecutter-ml-apigw-pytorch/{{cookiecutter.project_name}}/app/requirements.txt

```diff
@@ -0,0 +1,4 @@
+-f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
What is this for?

Reply:
PyTorch builds CPU-only wheels that are much smaller than the regular PyTorch (like 500MB vs 60MB or so iirc). Those are linked from that repo. It's the officially supported and documented way though so not a hack.

https://pytorch.org/get-started/locally/